cli/dts/lib.deno.unstable.d.ts

```diff
@@ -1396,6 +1396,23 @@ declare namespace Deno {
    */
   export function upgradeHttpRaw(request: Request): [Deno.Conn, Uint8Array];
 
+  /** The object that is returned from a {@linkcode Deno.upgradeWebSocket}
+   * request.
+   *
+   * @category Web Sockets */
+  export interface WebSocketUpgrade {
```

Comment:
It seems this is not used anywhere?

Reply:
it should overload the same named interface as in `lib.deno.ns.d.ts`